resources/migrations/001_update_migrations.yaml

```
@@ -11297,17 +11297,6 @@ databaseChangeLog:
       validCheckSum: ANY
       author: ranquild
       comment: Deduplicate metabase_database on (name, engine)
-      preConditions:
```

To not skip it on CI. Otherwise it can hide errors.